src/classes/default.js

```diff
@@ -100,7 +100,7 @@ var base = module.exports = Common.extend({
       var self = this;
       var appendTo = self._append;
       var append = prepend = '';
-      var scale = (self.props.scale || self.props.opts);
+      var scale = self.props.opts; // (self.props.scale || self.props.opts);
```

Comment:
ill need run some additional test to make sure this is applicable.  The current unit test does not cover some of the newly added svg api that may depend on that.